docs/troubleshooting.md

```diff
@@ -1,19 +1,66 @@
 # Troubleshooting
 
+Start by reviewing the [OpenTelemetry Collector troubleshooting
+documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/troubleshooting.md).
+
+## Gathering Support Information
+
+If you are unable to determine why something is not working then you can [email
+support](mailto:signalfx-support@splunk.com). When opening a support request,
```

Comment:
Should there be a stipulation to be an existing customer or is this open to all?

Reply:
Ah, good call -- must be a customer. I will update.